### PR TITLE
protodoc: Add label for field.

### LINKED ISSUE #3458: Documentation generated from .proto file doesn't specify if something is an array/repeated
For instance, from https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/cds.proto#cluster:

```
hosts
(core.Address) If the service discovery type is STATIC, STRICT_DNS or LOGICAL_DNS, then hosts is required.
```

It would be useful if the type said `repeated core.Address`, so similar.

### PR DESCRIPTION
**Pull request by mk46**
Signed-off-by: Manish Kumar <manish.kumar1@india.nec.com>
 
Commit Message: Add label for field in protobuf.
Additional Description: Add label for the field in protobuf. See #3458  
Risk Level: High
Testing: format
Fixes #3458

### CODE REVIEW

**Review by ggreenway — COMMENTED** [review]
I think this is a good improvement. At first glance, I found it hard to see/notice the new text. Maybe it should be bold? Or following the field-name in parentheses (instead of by the type). But I'm ok with starting with this and iterating later if people have a difference preference. @htuch any feedback?

File: tools/protodoc/protodoc.py [hunk 1]
```diff
@@ -498,9 +498,14 @@ def FormatFieldAsDefinitionListItem(outer_type_context, type_context, field, pro
         manifest_description.edge_config)
   else:
     formatted_security_options = ''
-
-  comment = '(%s) ' % ', '.join([FormatFieldType(type_context, field)] +
-                                field_annotations) + formatted_leading_comment
+  pretty_label_names = {
+      field.LABEL_OPTIONAL: '',
+      field.LABEL_REQUIRED: 'required ',
```
Comment (htuch) [inline tools/protodoc/protodoc.py:503]: I don't think this does anything, as all fields are technically required in proto3. Instead, we have https://github.com/envoyproxy/envoy/blob/master/tools/protodoc/protodoc.py#L458 for where PGV indicates that the field is required by annotation.

File: tools/protodoc/protodoc.py [hunk 2]
```diff
@@ -498,9 +498,14 @@ def FormatFieldAsDefinitionListItem(outer_type_context, type_context, field, pro
...
+      field.LABEL_REPEATED: 'repeated ',
```
Comment (htuch) [inline tools/protodoc/protodoc.py:504]: Yep, maybe add bold or italics or something to make it clear that this is distinguished.

**Review by htuch — CHANGES_REQUESTED** [review]
Thanks; agree with Greg that this is great but maybe we can add some formatting on the repeated..
/wait

File: tools/protodoc/protodoc.py (hunk 2 above)
Comment (mk46) [reply tools/protodoc/protodoc.py]: Added bold font.

File: tools/protodoc/protodoc.py (hunk 1 above)
Comment (mk46) [reply tools/protodoc/protodoc.py]: Removed required.

**Review by htuch — APPROVED** [review]
LGTM, thanks!

**Pull request merged**